### PR TITLE
Fix: Nav menu not scrolling to the appropriate CSS ID if Scroll Snap is enabled [ED-5439]

### PR DESCRIPTION
**Pull request by guyc-E**

### CODE REVIEW

File: assets/dev/js/frontend/utils/anchors.js [hunk 1]
```diff
@@ -84,7 +84,5 @@ module.exports = elementorModules.ViewModule.extend( {
 
 	onInit: function() {
 		elementorModules.ViewModule.prototype.onInit.apply( this, arguments );
-
-		this.bindEvents();
```
Comment (guyc-E) [inline assets/dev/js/frontend/utils/anchors.js:88]: This method is already triggered from ViewModule parent class.
It causes the events to trigger twice.

File: assets/dev/js/frontend/utils/anchors.js [hunk 2]
```diff
@@ -7,7 +7,7 @@ module.exports = elementorModules.ViewModule.extend( {
 			selectors: {
 				links: 'a[href*="#"]',
 				targets: '.elementor-element, .elementor-menu-anchor',
-				scrollable: 'html, body',
+				scrollable: isScrollSnapActive() ? 'body' : 'html, body',
```
Comment (guyc-E) [inline assets/dev/js/frontend/utils/anchors.js:10]: If there are two elements, it causes the scrolling animation to run twice and it sometimes breaks scroll-snap. In line 75

File: assets/dev/js/frontend/utils/anchors.js (hunk 1 above)
Comment (guyc-E) [reply assets/dev/js/frontend/utils/anchors.js:88]: Before I fixed the bug, the animate function in line 75 ran four times.

File: assets/dev/js/frontend/utils/anchors.js (hunk 2 above)
Comment (kobizz) [reply assets/dev/js/frontend/utils/anchors.js:10]: Is the `html` still needed for when there is no scroll snap?

**Review by kobizz — CHANGES_REQUESTED** [review]

File: assets/dev/js/frontend/utils/anchors.js (hunk 2 above)
Comment (guyc-E) [reply assets/dev/js/frontend/utils/anchors.js:10]: Yes, it works only with HTML, I thought maybe to delete the 'body' if scroll-snap is off, but I'm not sure if the 'html, body' together is fixing some sort of bug.

**Pull request merged**